File: LoraMeasurementsBackend/Model/Measurement.cs
namespace LoraMeasurementsBackend.Model;

public class Measurement
{
    public Guid Id { get;} = Guid.NewGuid();
    public DateTime MeasurementTime { get; set; }
    public double BarometricPressure { get; set; }
    public double RelativeHumidity { get; set; }
    public double Temperature { get; set; }
    public double AnalogIn { get; set; }
    public string DeviceId { get; set; }
    public double Rssi { get; set; }
    public double ChannelRssi { get; set; }
    public double Snr { get; set; }
    public int ChannelId { get; set; }
    public Device Device { get; set; }
}

File: LoraMeasurements/src/model/measurement.ts
export class Measurement {
    measurementTime: string;
    barometricPressure: number;
    relativeHumidity: number;
    temperature: number;
    analogIn: number;
    rssi: number;
    channelRssi: number;
    channelId: number;
    snr: number;


    constructor(measurementTime: string, barometricPressure: number, relativeHumidity: number,
                temperature: number, analogIn: number, rssi: number, channelRssi: number, 
                channelId: number, snr: number) {
        this.measurementTime = measurementTime;
        this.barometricPressure = barometricPressure;
        this.relativeHumidity = relativeHumidity;
        this.temperature = temperature;
        this.analogIn = analogIn;
        this.channelId = channelId;
        this.channelRssi = channelRssi;
        this.snr = snr;
        this.rssi = rssi;
    }
}